src/libraries/System.Runtime.Loader/tests/AssemblyExtensionsTest.cs

```diff
@@ -39,5 +39,18 @@ public static void ApplyUpdateInvalidParameters()
                     AssemblyExtensions.ApplyUpdate(typeof(AssemblyExtensions).Assembly, new ReadOnlySpan<byte>(metadataDelta), new ReadOnlySpan<byte>(ilDelta), ReadOnlySpan<byte>.Empty));
             }
         }
+
+        [Fact]
+        public void GetApplyUpdateCapabilitiesIsCallable()
+        {
+            var ty = typeof(System.Reflection.Metadata.AssemblyExtensions);
+            var mi = ty.GetMethod("GetApplyUpdateCapabilities", BindingFlags.NonPublic | BindingFlags.Static, Array.Empty<Type>());
+
+            Assert.NotNull(mi);
+
+            var result = mi.Invoke(null, null);
+
+            Assert.NotNull(result);
```

Comment:
It's bit odd to check for notnull if the return value is not declared as nullable

Reply:
Nullable annotations are just a suggestions. They do not guarantee any behavior. We have similar checks in many other tests.